spec/menu_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'menu'
+
+RSpec.describe "Menu" do
+
+  before(:each) do
+    @menu_contents = { "Fish" => 3, "Chips" => 2, "Veggie Burger" => 2, "Mushy Peas" => 0.5, "Curry Sauce" => 0.5, "Ketchup" => 0.2 }
+    @displayed_menu = "--Menu--\nFish: £3.0\nChips: £2.0\nVeggie Burger: £2.0\nMushy Peas: £0.5\nCurry Sauce: £0.5\nKetchup: £0.2\n"
```

Comment:
Why do you declare these at the top? I'd understand if they were used a number of times, but that is not the case. It feels less readable to have to always go up and down to know what is passed around when reading the tests.